### PR TITLE
8.0 'base' (onesteinbv)

### PR DESCRIPTION
**Pull request by legalsylvain**
Port 'base' modification realized by OneStein BV. (branch onesteinbv:8.0-os)

### CODE REVIEW

File: openerp/addons/base/migrations/8.0.1.3/post-migration.py [hunk 1]
```diff
@@ -43,7 +43,25 @@ def check_ir_actions_server_state(cr, pool):
                 ias.state, ias.id))
 
 
+def check_commercial_partner(cr, pool):
+    """ Test if the column is present (iow: if there exists a partner with a parent"""
+    if not openupgrade.column_exists(cr, "res_partner", "commercial_partner_id"):
+        sql = """ALTER TABLE res_partner ADD COLUMN commercial_partner_id int"""
+        cr.execute(sql)
```
Comment (pedrobaeza) [inline openerp/addons/base/migrations/8.0.1.3/post-migration.py]: Don't you need to fill the value?

File: openerp/addons/base/migrations/8.0.1.3/post-migration.py [hunk 2]
```diff
@@ -43,7 +43,25 @@ def check_ir_actions_server_state(cr, pool):
...
+
+
+def remove_account_report_company_record(cr, pool):
+    """An ir_ui_view record from the discontinued module account_report_company is not removed and causes an
+    AttributeError, remove it here to fix that error. """
+    view_obj = pool.get('ir.ui.view')
+    view_ids = view_obj.search(cr, SUPERUSER_ID,
```
Comment (pedrobaeza) [inline openerp/addons/base/migrations/8.0.1.3/post-migration.py]: This should be a search with the ID caught from ir.model.data reference.

File: openerp/addons/base/ir/ir_ui_view.py [hunk 3]
```diff
@@ -208,16 +208,7 @@ def _check_xml(self, cr, uid, ids, context=None):
...
-                    self.postprocess_and_fields(cr, uid, view.model, view_doc, view.id, context=context)
-                except Exception:
-                    # OpenUpgrade: We ignore all view errors as they are
-                    # caused by views introduced by models not year loaded
-                    _logger.warn(
-                        "Can't render view %s for model: %s. If you are "
-                        "migrating between major versions of OpenERP, "
-                        "this is to be expected (otherwise, do not run "
-                        "OpenUpgrade server).", view.xml_id, view.model)
+                self.postprocess_and_fields(cr, uid, view.model, view_doc, view.id, context=context)
```
Comment (pedrobaeza) [inline openerp/addons/base/ir/ir_ui_view.py]: What does this?

Comment (pankk) [inline openerp/addons/base/ir/ir_ui_view.py]: There was a new pull request merge 5 hours ago #111, which still needs to be backported.

Comment (pedrobaeza) [inline openerp/addons/base/ir/ir_ui_view.py]: It has been already merged. Do you still need this?

Comment (pankk) [inline openerp/addons/base/ir/ir_ui_view.py]: It needs to be merged on onestein 8.0 as well, that should omit this diff. So the changes here to ir_ui_view are not needed as merging these in would actually revert #111.

Comment (pedrobaeza) [inline openerp/addons/base/ir/ir_ui_view.py]: OK, a rebase is needed then.

File: openerp/addons/base/migrations/8.0.1.3/post-migration.py (hunk 1 above)
Comment (pankk) [inline openerp/addons/base/migrations/8.0.1.3/post-migration.py]: Sorry this needs to be reviewed. This was a workaround and might no longer be required with the merge of Odoo 8.0 (8ba452fc19382eb292bf1761e084ba2145e01b03)

Comment (legalsylvain) [inline openerp/addons/base/migrations/8.0.1.3/post-migration.py]: Removed.

File: openerp/addons/base/ir/ir_ui_view.py (hunk 3 above)
Comment (legalsylvain) [inline openerp/addons/base/ir/ir_ui_view.py]: Done.

File: openerp/addons/base/migrations/8.0.1.3/post-migration.py [hunk 4]
```diff
@@ -43,7 +43,22 @@ def check_ir_actions_server_state(cr, pool):
...
 
+def remove_account_report_company_record(cr, pool):
+    """An ir_ui_view record from the discontinued module """
+    """account_report_company is not removed and causes an
+    AttributeError, remove it here to fix that error. """
+    view_obj = pool['ir.ui.view']
+    try:
+        view_id = pool['ir.model.data'].get_object_reference(
+            cr, SUPERUSER_ID, 'account_report_company',
+            'account_report_copmany_partner_kanban_view')[1]
```
Comment (legalsylvain) [inline openerp/addons/base/migrations/8.0.1.3/post-migration.py:54]: PD: I'm not dyslexic, this is the real name of the view. ;-)

**Comment by hbrunn** [comment]
:+1:

**Comment by paulius-sladkevicius** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**